.circleci/config.yml

```diff
@@ -41,7 +41,7 @@ jobs:
           paths:
             - "~/.cargo"
             - "./target"
-  benchmark-build:
+  benchmark:
     docker:
       - image: polymathnet/rust:debian-nightly-2020-09-28
```

Comment:
Can the image be made variable based on the toolchain file in the repo?  Concourse reads the file into a variable that then gets used in the pipeline to pick the image.

Reply:
Not without a lot of hacking. There's no code or shell available before the docker image is loaded.